src/components/OrderBookChart.stories.tsx

```diff
@@ -1,256 +1,47 @@
-import React, { useMemo } from 'react'
+import React from 'react'
 // also exported from '@storybook/react' if you can deal with breaking changes in 6.1
 import { Story, Meta } from '@storybook/react/types-6-0'
 
 import OrderBookChart, { OrderBookChartProps } from './OrderBookChart'
-import { Network, TokenDetails } from 'types'
-import realData from 'storybook/USDC-DAI_OrderBook_sample.json'
-import sampleDataSet from 'storybook/orderbookSamples'
+import USDCxDAIdata from 'storybook/USDC-DAI_OrderBook_sample.json'
 import { processRawApiData } from './OrderBookWidget'
-import { OrderBookData } from 'api/dexPriceEstimator/DexPriceEstimatorApi'
+import {
+  networkIds,
+  defaultNetworkId,
+  defaultBaseToken,
+  defaultQuoteToken,
+  findTokenBySymbolOrAddress,
+} from 'storybook/tokenDefaults'
 
-type SampleData = typeof sampleDataSet[0]
-
-const networkIds = Object.values(Network).filter(Number.isInteger)
-
-const defaultNetworkId = Network.Mainnet
-// All Default Tokens
-const tokenList: TokenDetails[] = CONFIG.initialTokenList.map(
-  ({ id, name, symbol, addressByNetwork, decimals = 18 }) => ({
-    id,
-    name,
-    symbol,
-    address: addressByNetwork[defaultNetworkId] || '0x',
-    decimals: decimals,
-  }),
-)
-
-// Default params to be used as initial values
-// and when there's no Token found for symbol
 const defaultParams: Omit<OrderBookChartProps, 'data'> = {
-  baseToken: {
-    id: 1,
-    name: 'Base Token',
-    symbol: 'BASE',
-    address: '0x1',
-    decimals: 18,
-  },
-  quoteToken: {
-    id: 2,
-    name: 'Quote Token',
-    symbol: 'QUOTE',
-    address: '0x2',
-    decimals: 18,
-  },
+  baseToken: defaultBaseToken,
+  quoteToken: defaultQuoteToken,
   networkId: defaultNetworkId,
 }
 
-// Token symbols to use in control selector
-const tokenSymbols = [defaultParams.baseToken, defaultParams.quoteToken, ...tokenList].map(
-  (token) => token.symbol || token.address,
-)
-
 export default {
-  title: 'Components/OrderBookChart',
+  title: 'OrderBook/Real Data',
   component: OrderBookChart,
   argTypes: {
     networkId: { control: { type: 'inline-radio', options: networkIds } },
-    baseToken: {
-      control: { type: 'select', options: tokenSymbols },
-    },
-    quoteToken: {
-      control: { type: 'select', options: tokenSymbols },
-    },
+    baseToken: { control: null },
+    quoteToken: { control: null },
     data: { control: null },
   },
   decorators: [(Story): JSX.Element => <div style={{ height: '97vh' }}>{Story()}</div>],
 } as Meta
 
-//-------------------REAL-DATA------------------
-
 // Template for real data
 // fetched from price-estimation service
 const RealTemplate: Story<OrderBookChartProps> = (args) => <OrderBookChart {...args} />
 
-const DAI = tokenList.find((token) => token.symbol === 'DAI')
-const USDC = tokenList.find((token) => token.symbol === 'USDC')
+const DAI = findTokenBySymbolOrAddress('DAI', defaultBaseToken)
```

Comment:
data is preprocessed here, unlike in samples, so changing tokens will only change the text, not recalculate based on decimals
so not much use
and this data only works for these tokens (or decimals) for others it will look bad